src/app/css/_collapsibleSection.scss

```diff
@@ -0,0 +1,16 @@
+/***********************************************************
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License
+ **********************************************************/
+ @import 'themes';
+ 
+ .collapsible-section {
+    padding-bottom: 20px;
+    @include themify($themes) {
+        border: 1px solid themed('borderColor')
+    }
+    .collapsible-section-icon {
+        float: left;
+        margin-left: -5px;
```

Comment:
I'd recommend going with something more like this to avoid button rendering beyond the .collapsible-section div.

```
margin-left: 0px;
margin-right: 4px;
padding-left: 0px;
padding-right: 0px;
```

Reply:
+1 I updated margin-right to add some spacing between the button and the label but chose to leave the standard padding as-is since it wasn't causing issues.